src/test/mir-opt/const_allocation/rustc.main.ConstProp.after.mir

```diff
@@ -4,85 +4,65 @@
-        StorageLive(_2);                 // bb0[1]: scope 0 at $DIR/const_allocation.rs:6:5: 6:8
-        _2 = const Scalar(alloc0+0): &&[(std::option::Option<i32>, &[&str])]; // bb0[2]: scope 0 at $DIR/const_allocation.rs:6:5: 6:8
+        StorageLive(_1);                 // bb0[0]: scope 0 at $DIR/const_allocation.rs:8:5: 8:8
+        StorageLive(_2);                 // bb0[1]: scope 0 at $DIR/const_allocation.rs:8:5: 8:8
+        _2 = const Scalar(alloc0+0): &&[(std::option::Option<i32>, &[&str])]; // bb0[2]: scope 0 at $DIR/const_allocation.rs:8:5: 8:8
                                          // ty::Const
                                          // + ty: &&[(std::option::Option<i32>, &[&str])]
                                          // + val: Value(Scalar(alloc0+0))
                                          // mir::Constant
-                                         // + span: $DIR/const_allocation.rs:6:5: 6:8
+                                         // + span: $DIR/const_allocation.rs:8:5: 8:8
                                          // + literal: Const { ty: &&[(std::option::Option<i32>, &[&str])], val: Value(Scalar(alloc0+0)) }
-        _1 = (*_2);                      // bb0[3]: scope 0 at $DIR/const_allocation.rs:6:5: 6:8
-        StorageDead(_2);                 // bb0[4]: scope 0 at $DIR/const_allocation.rs:6:8: 6:9
-        StorageDead(_1);                 // bb0[5]: scope 0 at $DIR/const_allocation.rs:6:8: 6:9
-        _0 = ();                         // bb0[6]: scope 0 at $DIR/const_allocation.rs:5:11: 7:2
-        return;                          // bb0[7]: scope 0 at $DIR/const_allocation.rs:7:2: 7:2
+        _1 = (*_2);                      // bb0[3]: scope 0 at $DIR/const_allocation.rs:8:5: 8:8
+        StorageDead(_2);                 // bb0[4]: scope 0 at $DIR/const_allocation.rs:8:8: 8:9
+        StorageDead(_1);                 // bb0[5]: scope 0 at $DIR/const_allocation.rs:8:8: 8:9
+        _0 = ();                         // bb0[6]: scope 0 at $DIR/const_allocation.rs:7:11: 9:2
+        return;                          // bb0[7]: scope 0 at $DIR/const_allocation.rs:9:2: 9:2
     }
 }
 
-alloc0 {
-    size: 16
-    alignment: 8
-    static const_allocation[317d]::FOO[0]
-    memory: ├──────alloc17+0──────┤ 03 00 00 00 00 00 00 00 
+alloc0 (const_allocation[317d]::FOO[0]) size: 16, align: 8 {
+    ├──────alloc17+0──────┤ 03 00 00 00 00 00 00 00 
 }
 
-alloc17 {
-    size: 72
-    alignment: 8
-    memory: 00 00 00 00 __ __ __ __ ├──────alloc4+0───────┤ 00 00 00 00 00 00 00 00 00 00 00 00 __ __ __ __ ├──────alloc8+0───────┤ 02 00 00 00 00 00 00 00 01 00 00 00 2a 00 00 00 ├──────alloc13+0──────┤ 03 00 00 00 00 00 00 00 
+alloc17 size: 72, align: 8 {
+    0x00 │ 00 00 00 00 __ __ __ __ ├──────alloc4+0───────┤ 00 00 00 00 00 00 00 00 00 00 00 00 __ __ __ __ 
+    0x20 │ ├──────alloc8+0───────┤ 02 00 00 00 00 00 00 00 01 00 00 00 2a 00 00 00 ├──────alloc13+0──────┤ 
```

Comment:
This still exceeds 100 characters, so the lines are still pretty long. Also it took me a bit to realize that the thing at the beginning is the offset. And it's inconsistent to only have it in the multi-line case.

Reply:
It's super useless in the single line case. I made it inconsistent on purpose. Having `0x0 | 42` is just bad imo

Reply:
Fair, though I am not entirely convinced.

But still, currently it's confusing. Maybe add a leading "+" to get some similarity with `alloc22+14`?

Reply:
I changed my settings so this doesn't line-wrap any more, and now it actually looks pretty good.

So the problem, if any, is that it takes more than 100 characters. 32 bytes with 3 characters each are 96, so with indentation it takes 99 characters which barely fits, but with the `0x00 |` thing it becomes too long. :/
Maybe just go for 16 bytes per line?